test/Microsoft.IdentityModel.Tokens.Tests/CryptoProviderCacheTests.cs

```diff
@@ -591,6 +608,8 @@ public class CryptoProviderCacheTheoryData : TheoryDataBase
 
     public class InMemoryCryptoProviderCachePublic : InMemoryCryptoProviderCache
     {
+        public bool DisposeCalled { get; set; } = false;
```

Comment:
this should also be in a #if block if it only gets used in a single target

Reply:
InMemoryCryptoProviderCache now implements IDisposable on all targets, so actually it is the #if block around the Dispose() method that should be removed. Fix made here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1605/commits/af3243229ca3f4885398299f0d733066d3b9be46.